modules/apps/client-extension/client-extension-web/src/main/java/com/liferay/client/extension/web/internal/constants/ClientExtensionAdminWebKeys.java

```diff
@@ -29,4 +29,11 @@ public class ClientExtensionAdminWebKeys {
 		EDIT_CLIENT_EXTENSION_ENTRY_PART_DISPLAY_CONTEXT =
 			"EDIT_CLIENT_EXTENSION_ENTRY_PART_DISPLAY_CONTEXT";
 
+	public static final String VIEW_CLIENT_EXTENSION_ENTRY_DISPLAY_CONTEXT =
+		"VIEW_CLIENT_EXTENSION_ENTRY_DISPLAY_CONTEXT";
+
+	public static final String
+		VIEW_CLIENT_EXTENSION_ENTRY_PART_DISPLAY_CONTEXT =
```

Comment:
I've just seen that we're keeping the naming pattern with `EDIT_CLIENT_EXTENSION_ENTRY_PART_DISPLAY_CONTEXT`, nevertheless, naming looks confusing to me as the "part" does not refer to a particular field/item in the CET but to the specific part not present in CET.java. So, the naming looks not telling us the "specificity" meaning of the display context object. I'm ok with keeping the pattern though

Reply:
That's the same pattern we use in the edit JSP. The only difference here is that we are providing a default `view.jsp` file but since it can be overridden, if the developer decided to do so, the structures of edit and view would be identical.